Makefile

```diff
@@ -160,9 +160,14 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	rm Dockerfile.cross
 
 ##@ Deployment
+## settings parameter for deployment 
+
+# getting the cloud id and the IngressHost 
+
+CLOUD_ID := $(shell oc get clusterversion -o jsonpath='{.items[].spec.clusterID}{"\n"}')
 
 ifndef ignore-not-found
-  ignore-not-found = false
+  ignore-not-found = true
```

Comment:
This should be removed before merging... and can be specified at the command line like this:

```
make IMAGE_TAG_BASE=<your repo name here> undeploy docker-push install deploy ignore-not-found=true
```